functions/indicatorScoreCalculation/over65.R

```diff
@@ -0,0 +1,55 @@
+# 
+# data <- allData
+# geometry <- geometryFiles[[1]]
+# name <- names(geometryFiles)[[1]]
+
+processOver65 <- function(geometry, name, data){
+  # select the data set of interest 
+  vals <- data[[grep(pattern = name, x = names(data))]] |> as.data.frame()
+  
+  # structure then generate and select measures of concern
+  output <- structureACS(vals) |>
+    dplyr::mutate(
+      age_over65 = sum(B01001_020, B01001_021))|>
```

Comment:
Double-checking this - there are only two variables being summed here while 12 different variables were downloaded from the ACS for over 65.

Reply:
thanks for the catch, 
included the range of indicators for both male 20:25 and female 44:49